pkg/converter/convert_test.go

```diff
@@ -210,6 +212,15 @@ func TestConvert(t *testing.T) {
 				tenantID,
 			},
 		},
+		{
+			name:         "isExecUsingkubelogin",
```

Comment:
the test case name should be descriptive

Reply:
also, please add the rest of test cases.

Reply:
can you also update the previous test cases to indicate they are testing against legacy azure auth? like `with legacy azure auth kubeconfig, ...`